### ISSUE TITLE
Codecrunch Multiple Output Detected

### ISSUE THREAD

**Issue by geraldckj** [nus-cs2030/2223-s2#35]
# Summary

Hi, codecrunch throws me a compilation error when submitting my lab1. The error is "Multiple output detected"

## Description:
Test cases seem to pass on my local machine with no issues.
However, I did use some println statements to achieve the output from the program.

## Screenshots

![image](https://user-images.githubusercontent.com/56596389/215306346-8103224d-e599-4e65-b844-b0de84277f43.png)

**Comment by lihongguang00** [comment]
Hi gerald,

You can check out this post to resolve this issue:

https://github.com/nus-cs2030/2223-s2/issues/27#issue-1559803778

Cheers

**Comment by SimonSeeYongsheng** [comment]
I don't think you are supposed to be printing anything, as the Main.java class does that for you already.

**Comment by rshalinisai** [comment]
I had this problem too as I had multiple print statements but we are only supposed to return not print
